docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - DOCUMENTSTORE_PARAMS_HOST=elasticsearch
       - PIPELINE_YAML_PATH=/opt/pipelines/pipelines.haystack-pipeline.yml
       - TOKENIZERS_PARALLELISM=false
+#      - GUNICORN_CMD_ARGS="--timeout=3000"
```

Comment:
```suggestion
# Uncomment the following line to customise how much time a worker can spend serving a request before it times out
#      - GUNICORN_CMD_ARGS="--timeout=3000"
```

Reply:
Done. :) Sorry I think I just added it without comment because maybe my brain wasn't fully awake yet :grimacing: